modules/ROOT/pages/getting-started.adoc

```diff
@@ -1,6 +1,14 @@
 :experimental:
 = Fedora CoreOS - Getting Started
 
+== Introduction
+
+include::update-streams-info.adoc[leveloffset=2]
```

Comment:
Hmm, do we want all that text in the Getting Started page? How about something more brief and a link to the page? Something like:

> There are three Fedora CoreOS (FCOS) update streams available: `stable`, `testing`, and `next`. In general, you will want to use `stable`, but it is recommended to run some machines on `testing` and `next` as well. For more information on streams and switching between them, see xref:update-streams.adoc[Update Streams].

WDYT?

Reply:
I agree. Specifically, I think the `update-streams-info` page belongs to the "Reference Pages" section.

Reply:
Done

Reply:
> I agree. Specifically, I think the `update-streams-info` page belongs to the "Reference Pages" section

Why does it not fit under the "OS updates" section in the navbar?

Reply:
That sounds fine too, :+1:.